server/test/unit/plugins/util/test_verification.py

```diff
@@ -49,6 +36,10 @@ def test_SHA256_to_sha256(self):
 
         self.assertEqual(checksum_type, 'sha256')
 
+    def test_invalid_type_raises_coded_exception(self):
+        self.assertRaises(PulpCodedException,
+                          verification.sanitize_checksum_type, 'not_a_real_checksum')
```

Comment:
You're going to be sorry when my new checksum algorithm, 'not_a_real_checksum', becomes widely accepted and breaks this unit test.